stock_release_channel_cutoff/views/stock_release_channel_views.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="stock_release_channel_form_view" model="ir.ui.view">
+        <field name="name">stock.release.channel.form.inherited</field>
+        <field name="model">stock.release.channel</field>
+        <field
+            name="inherit_id"
+            ref="stock_release_channel.stock_release_channel_form_view"
+        />
+        <field name="arch" type="xml">
+            <field name="release_forbidden" position="after">
```

Comment:
Move field above end time as this is a setting and not an option
![image](https://github.com/OCA/wms/assets/7802725/e0cea460-9cbd-4f0d-bb66-2c849636486c)